index.html

```diff
@@ -193,20 +194,27 @@ <h2 class="conta">CONTACT US</h2>
     if you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it
 </p>
 
-<form >
-    <input class="text-inp" type="text" placeholder=" Text Input">
-
-    <input class="email" type="text" placeholder=" Jeremy@gmail.com">
-
-    <textarea class="send-text" cols="30" rows="10" placeholder="
-Enter text here"></textarea>
-</form>
-
-<button type="button" class="btn-sec"> See Project </button>
-
-<div class="cont-line">.</div>
-
-
+    <form action="https://formspree.io/f/mlezygyr" method="post">
+        <ul class="input-list">
+         <li>
+           <label for="name">Name:</label>
+           <input type="text" id="name" name="user_name" placeholder="  Name" maxlength="30" required>
+         </li>
+         <li>
+           <label for="mail">E-mail:</label>
+           <input type="email" id="mail" name="user_email" placeholder="  E-mail" required>
+         </li>
+         <li>
+           <label for="msg">Message:</label>
+           <textarea cols="33" rows="16"  id="msg" name="user_message" 
+           placeholder=" 
+ Enter text here" maxlength="500" required ></textarea>
+         </li>
+         <li class="button">
+           <button type="submit" id="bttn"> see project </button>
+         </li>
```

Comment:
- [ ] In the design, there are no labels for the form elements because there is a placeholder text for each element. Kindly adjust to the design.

#### Design
![image](https://user-images.githubusercontent.com/44057863/177427565-df58e462-4674-4876-a666-b5319959a8c7.png)

#### Impl
![image](https://user-images.githubusercontent.com/44057863/177427518-974d5ab5-1897-4870-9ea1-6009c0a243d6.png)